Monika After Story/game/definitions.rpy

```diff
@@ -6099,23 +6099,36 @@ init -1 python:
         for savegame in renpy.list_saved_games(fast=True):
             renpy.unlink_save(savegame)
 
-
-    def delete_character(name):
+    def mas_delete_chr(name, log=False):
         """
-        Deletes a .chr file for a character
+        Deletes a .chr file
 
         IN:
-            name of the character who's chr file we want to delete
+            name - str - filename
         """
-        if persistent.do_not_delete:
-            return
-
         try:
-            os.remove(config.basedir + "/characters/" + name + ".chr")
+            if not name.endswith(".chr"):
+                name += ".chr"
+            os.remove(
+                os.path.join(config.basedir, "characters", name)
+            )
 
-        except:
-            pass
+        except Exception as e:
+            if log:
+                store.mas_utils.mas_log.error(
+                    "Failed to delete a '{}' chr file: {}".format(name, e)
+                )
```

Comment:
I feel like this is an additional function which won't really be used outside `mas_delete_all_chrs` as we never meddle with chr files in MAS.

Reply:
@dreamscached can you do the req changes?

Reply:
Yes, of course!

Reply:
So, what exactly I should do about it? Should I remove this func?

Reply:
1. remove the generalised function
2. keep the func that removes all `chr` files, but make it use dock station

Reply:
Done!